WISE-Community/WISE-Client
# feat(Tag): Use inline editing instead of popup dialogs to edit tags

## Changes
- Create tag and edit tag now use inline editing instead of popup dialogs

## Test
- Make sure creating a tag still works
- Make sure editing a tag still works
